src/event.rs

```diff
@@ -248,9 +248,9 @@ impl Event {
         }
     }
 
-    pub fn set_screen<'a, T: Into<Option<&'a Screen>>>(&mut self, screen: T) {
+    pub fn set_screen(&mut self, screen: Option<&Screen>) {
```

Comment:
These should also all be `IsA<_>` or not?

Reply:
Seems it final
https://github.com/gtk-rs/gdk/blob/47bdccab08a8366788b74e2ab1bc97f6f7269d81/src/auto/app_launch_context.rs#L46

Reply:
That's good

Reply:
Should double-check for all these types then :)